lms/templates/help_modal.html

```diff
@@ -18,194 +18,207 @@
   <a href="#help-modal" rel="leanModal" role="button">${_("Support")}</a>
 </div>
 
-<section id="help-modal" class="modal" aria-hidden="true" role="dialog" tabindex="-1" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
-  <div class="inner-wrapper">
+<section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
+  <div class="inner-wrapper" id="help_wrapper">
     ## TODO: find a way to refactor this
-    <button class="close-modal" tabindex="0">
+    <button class="close-modal "tabindex="0">
```

Comment:
I did not hand-edit these reverts. I simply did the git command to revert a commit, so the new code is what was there before (for better or worse!).